### ISSUE TITLE
Incorrect attribute assignment for multiple inline code blocks in a paragraph

### ISSUE THREAD

**Issue by rivy** [arve0/markdown-it-attrs#52]
These tests fail...

```javascript
it('should work with multiple inline code blocks in same paragraph', () => {
  src = 'bla `click()`{.c} blah `release()`{.cpp}';
  expected = '<p>bla <code class="c">click()</code> blah <code class="cpp">release()</code></p>\n';
  assert.equal(md.render(src), expected);
});

it('should work with interleaved inline code blocks in same paragraph #1', () => {
  src = 'bla `click()` blah `release()`{.cpp} foo `repeat()`{.c}';
  expected = '<p>bla <code>click()</code> blah <code class="cpp">release()</code> foo <code class="c">repeat()</code></p>\n';
  assert.equal(md.render(src), expected);
});

it('should work with interleaved inline code blocks in same paragraph #2', () => {
  src = 'bla `click()` blah `release()`{.cpp} foo `repeat()`{.c} bar `end()`';
  expected = '<p>bla <code>click()</code> blah <code class="cpp">release()</code> foo <code class="c">repeat()</code> bar <code>end()</code></p>\n';
  assert.equal(md.render(src), expected);
});
```

**Comment by arve0** [comment]
Thanks for reporting! :octocat: 

Fixed in `v1.2.1`.

**Comment by rivy** [comment]
Thanks! I appreciate the quick fix, and the module itself.

If you're interested, I'm using it in a user script for local markdown rendering ([`markdown-render`](https://github.com/rivy/js-user.markdown-render)).
